frontend_tests/node_tests/lightbox.js

```diff
@@ -13,29 +13,24 @@ set_global('overlays', {
 set_global('popovers', {
     hide_all: () => {},
 });
-set_global('$', function () {
-    return {
-        hasClass: () => false,
-        closest: () => [],
-        attr: (attr) => attr,
-        parent: () => ({
-            closest: () => ({
-                attr: (attr) => attr,
-            }),
-            attr: (attr) => attr,
-        }),
-        html: () => ({
-            show: () => {},
-        }),
-        hide: () => {},
-        show: () => {},
-        text: () => '',
-    };
-});
+const { JSDOM } = require('jsdom');
+const { window } = new JSDOM('<!DOCTYPE html><p>Hello world</p><div class="image-actions"><a class="open"></a></div>');
```

Comment:
We don't generally use jsdom in our tests because it's super slow; we usually usually zjsquery instead.  Did you look at doing so?  

https://github.com/zulip/zulip/blob/master/frontend_tests/node_tests/zjquery.js may be helpful.